nekoyume/Assets/_Scripts/UI/Shop/ShopBuy.cs

```diff
@@ -307,56 +275,52 @@ private void OnClickShopItem(ShopItemView view)
 
         private void CleanUpWishListAlertPopup(System.Action callback)
         {
-            if (shopItems.SharedModel.isMultiplePurchase && shopItems.SharedModel.wishItems.Count > 0)
+            if (shopItems.SharedModel.isMultiplePurchase && shopItems.SharedModel.WishItemCount > 0)
             {
                 Widget.Find<TwoButtonPopup>().Show(L10nManager.Localize("UI_CLOSE_BUY_WISH_LIST"),
-                    L10nManager.Localize("UI_YES"),
-                    L10nManager.Localize("UI_NO"),
-                    callback);
+                    L10nManager.Localize("UI_YES"), L10nManager.Localize("UI_NO"), callback);
             }
             else
             {
                 callback.Invoke();
             }
         }
 
-        public static PurchaseInfo GetPurchseInfo(ShopItem shopItem)
-        {
-            return new PurchaseInfo(shopItem.ProductId.Value,
-                shopItem.SellerAgentAddress.Value,
-                shopItem.SellerAvatarAddress.Value,
-                shopItem.ItemSubType.Value,
-                shopItem.Price.Value);
-        }
-
-        public static ItemBase GetItemBase(Buy.PurchaseResult result)
+        public static PurchaseInfo GetPurchseInfo(Guid orderId)
         {
-            if (result.itemUsable != null)
-            {
-                return result.itemUsable;
-            }
-
-            if (result.costume != null)
-            {
-                return result.costume;
-            }
-
-            return (ItemBase)result.tradableFungibleItem;
+            var order = Util.GetOrder(orderId);
+            return new PurchaseInfo(orderId, order.TradableId, order.SellerAgentAddress,
+                order.SellerAvatarAddress, order.ItemSubType, order.Price);
         }
 
-        public static ItemBase GetItemBase(AttachmentActionResult result)
-        {
-            if (result.itemUsable != null)
-            {
-                return result.itemUsable;
-            }
-
-            if (result.costume != null)
-            {
-                return result.costume;
-            }
-
-            return (ItemBase)result.tradableFungibleItem;
-        }
+        // public static ItemBase GetItemBase(Buy.PurchaseResult result)
```

Comment:
Is this a necessary comments?

Reply:
It is correct to delete the legacy functions. 👍🏻